openstack/networking/v2/extensions/security/rules/requests.go

```diff
@@ -74,6 +74,7 @@ const (
 	ProtocolUDP       RuleProtocol  = "udp"
 	ProtocolUDPLite   RuleProtocol  = "udplite"
 	ProtocolVRRP      RuleProtocol  = "vrrp"
+	ProtocolANY       RuleProtocol  = "any"
```

Comment:
Not trying to be picky but wouldn't it make sense to call it `ProtocolAny`? Since "Any" isn't an acronym, like UDP or VRRP are.
What do you think?

Reply:
Updated to `ProtocolAny`, still this ANY is was acronym to "None" in the OpenStack module - not so clear logic...